partner_default_sale_discount/models/sale_order.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from lxml import etree
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    default_sale_discount = fields.Float(
+        related="partner_id.commercial_partner_id.default_sale_discount",
+        string="Default sales discount (%)",
+    )
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        """Inject the default in the context of the line this way for
+        making it inheritable.
+        """
```

Comment:
OK, then please add a comment to explain that, so next people reviewing this don't refactor it.